.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "src/llvm"]
 	path = src/llvm
-	url = https://github.com/rust-lang/llvm.git
+	url = https://github.com/alexcrichton/llvm.git
```

Comment:
Yes, I will do that after r+. This is currently necessary to pull in https://github.com/alexcrichton/llvm/commit/fd2b33b0d036814e4913ac630fb186fce1ca6df7